parallaxinc/Parallax-IDE
# Flux editor

#### What's this PR do?

Moves the editor and keypress bindings into Alt Flux.
#### Where should the reviewer start?

Look at the new actions and stores for editor and keys. Look at the key-extensions to see how keypresses are now registered using Flux actions.
#### How should this be manually tested?

Test the editor to ensure that you can still type text, save files, etc.
Test the six keypresses that are present in the keys Flux Store. All should function as they did before.
#### Any background context you want to provide?

Bringing basic editor and keypress functionality into Flux using Alt.
#### What are the relevant tickets?

References #115
